learningequality/studio
# Update makemessages & transfercontext scripts to use kolibri-tools CLI

## Summary
### Description of the change(s) you made

`kolibri-tools` relies now on using a CLI to run the commands needed for i18n message extraction and context transfer.

This provides commands that ought to properly extract all messages for the frontend and includes an updated command for context transfer.

One thing I wondered is if we should remove the `combineprofiles` script as that doesn't seem to be a part of `kolibri-tools/i18n` any longer.

### Manual verification steps performed
1. Run `make i18n-upload --branch=something` and see that the latest messages have been uploaded on Crowdin (or, if you're @bjester test it out on unstable :smile: )
2. Run `make i18n-transfer-context` and it should run, then lint, then things should be updated (assuming more context has been added on Crowdin that isn't currently in the code, anyway).
